Create ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md (#2689)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-11-1-bulk-operation-scheduling-delay-in-milliseconds-not-working-7607-graft-release.md

---
date: ""

> title: Bulk operation scheduling delay in milliseconds now properly

Reviewer: ```suggestion
title: Bulk operation scheduling delay in milliseconds now works properly
```

> title: Bulk operation scheduling delay in milliseconds is now working

Reviewer: ```suggestion
title: Bulk operation scheduling delay in milliseconds now properly
```

> title: Bulk operation scheduling delay in milliseconds now works properly

Reviewer: ```suggestion
title: Scheduling bulk operations with a precise delay now works properly
```

> title: bulk operation scheduling delay in milliseconds not working (#7607) [GRAFT][release/cd] (#7677)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Bulk operation scheduling delay in milliseconds is now working
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4183
version: 1021.11.1
---

> bulk operation scheduling delay in milliseconds not working (#7607) [GRAFT][release/cd] (#7677)

Author: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT platform, bulk operations allow performing actions on multiple devices simultaneously. However, the scheduling delay in milliseconds for these operations was not working as expected. This issue has now been resolved. With this fix, bulk operations can be scheduled with a delay specified in milliseconds, providing more precise control over when the operations are executed on the target devices. This improvement ensures that bulk operations are carried out according to the specified delay, enhancing the reliability and efficiency of device management tasks.
```

Reviewer 2: Maybe shorter?

Scheduling bulk operations with delays in milliseconds has been fixed. Operations can now be scheduled with precise delays, improving the reliability and efficiency of device management tasks.

> In the Cumulocity IoT platform, bulk operations allow performing actions on multiple devices simultaneously. However, the scheduling delay in milliseconds for these operations was not working as expected. This issue has now been resolved. With this fix, bulk operations can be scheduled with a delay specified in milliseconds, providing more precise control over when the operations are executed on the target devices. This improvement ensures that bulk operations are carried out according to the specified delay, enhancing the reliability and efficiency of device management tasks.

Reviewer: ```suggestion
In the {{< product-c8y-iot >}} platform, bulk operations allow performing actions on multiple devices simultaneously. However, scheduling these operations with a delay in milliseconds was not working as expected. This issue has now been resolved. With this fix, bulk operations can now be scheduled with precise delays specified in milliseconds, enhancing the reliability and efficiency of device management tasks.
```
